ph5/utilities/tests/test_metadatatoph5.py

```diff
@@ -51,7 +50,7 @@ def test_main(self):
         test main function
         """
         testargs = ['metadatatoph5', '-n', 'master.ph5', '-f',
-                    'ph5/test_data/metadata/station.xml']
+                    '../metadata/station.xml']
```

Comment:
This works because it passes the check. And actually when users run ph5 commands, they always use relative path. You can look in create_ph5.sh as an example.

I don't use os.join() because, as I told you, it creates the path that's greater than 32 character length in das_t's 'raw_file_name_s' and will be chop off at the end.

Reply:
ok.. As this is an exception for some code that should be more robust. Put a comment stating so

Reply:
Sorry. You are correct at this point. I misread this.